R/autosyn.R

```diff
@@ -726,3 +753,75 @@ extract_ntpredictions.neuron <- function(x,
   }
 }
 
+# Get synapses as FlyWire annotations
+#' @export
+#' @rdname flywire_neurons_add_synapses
+flywire_synapse_annotations <- function(x,
+                                        file = NULL,
+                                        scale = 1/c(4,4,40), # from nm to voxel space
```

Comment:
I think `voxdims` may be better (because otherwise one has to think about which direction the scale factor is going ...)